docs/src/database.rst

```diff
@@ -66,12 +66,12 @@ Atlas MongoDB
 6. Add the IP of your compuer to the whitelist or "Allow access from anywhere."
 7. Click on "Connect your application".
 8. Orion supports MongoDB drive 3.4, so choose driver 3.4.
-9. Copy the generated SRV address and replace "USERNAME" and "PASSWORD" with your Atals MongoDB username and password.
+9. Copy the generated SRV address and replace "USERNAME" and "PASSWORD" with your Atlas MongoDB username and password.
 10. Test with::
 
-   mongo "mongodb+srv://orion-efjp0.mongodb.net/test" --username YOUR_USER_NAME
+   mongo [your uri] --username YOUR_USER_NAME
```

Comment:
Oh sorry I send messages to quickly. There is YOUR_USER_NAME so we should make it coherent and use YOUR_URI. Also, there is no step to let the user know how to get this URI for commandline call with `mongo`. Could you add it?

Reply:
I see. But I don't know how to get that URI either :sweat_smile:

Reply:
Oh, no problem! That's a good sign that it will be valuable to have this info in the doc. 
It is in Connect/Connect with the Mongo Shell.

Reply:
Thanks!